phpBB/phpbb/install/console/command/install/config/show.php

```diff
@@ -96,7 +96,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
 		try
 		{
-			$config = Yaml::parse(file_get_contents($config_file), true);
+			$config = Yaml::parse(file_get_contents($config_file));
```

Comment:
Acording to https://github.com/symfony/symfony/blob/4.4/UPGRADE-4.0.md

Removed support for passing true/false as the second argument to the parse() method to trigger exceptions when an invalid type was passed.

Before:

```php
Yaml::parse('{ "foo": "bar", "fiz": "cat" }', true);
```
After:
```php
Yaml::parse('{ "foo": "bar", "fiz": "cat" }', Yaml::PARSE_EXCEPTION_ON_INVALID_TYPE);
```